src/wrapper.jl

```diff
@@ -468,12 +534,16 @@ end
 function LGBM_BoosterCalcNumPredict(bst::Booster, num_row::Integer, predict_type::Integer,
                                     num_iteration::Int)
     out_len = Ref{Int64}()
+    start_iteration = Cint(1)
```

Comment:
should have added it as a param instead and use as 0 like in `LGBM_BoosterSaveModel`